json_schemas/suppliers.json

```diff
@@ -38,7 +38,6 @@
   },
   "required": [
     "id",
-    "name",
-    "contactInformation"
```

Comment:
Hmm... this is tricky.  We do want to enforce that new suppliers must provide some contact information.  But I can see that for updates we don't want to have to bundle it in every time.